src/options/popup.js

```diff
@@ -70,7 +87,31 @@ export default {
         </li>`
       ).join("")
       + '</ul>',
-    save: `<button class="cc-btn cc-save">Save</button>`
+    save: `<button class="cc-btn cc-save">Save</button>`,
+    customize: `<button class="cc-btn cc-customize">{{customize}}</button>`,
+    customizeHeader: `<div class="cc-customize-header">{{customizeHeader}}</div>`,
+    customizeMessage: `<div class="cc-customize-message">{{customizeMessage}}</div>`,
+    acceptSelected: `<div class="cc-compliance">
+        <span class="cc-accept-selected">
+          <button class="cc-btn cc-save">{{acceptSelected}}</button>
+        </span>
+      </div>`,
+    customizeCategories: `<ul class="cc-customize-categories">` +
+      CATEGORIES.map(category =>
+        `<li class="cc-category ${category}">
+            <label>
+              <input type="checkbox" id="${category}" name="${category}"/>
+              <span class="cc-btn-checkbox">${categoriesDisplayNames[category]}</span>
+            </label>
+            <label class="cc-readmore" for="readmore-${category}">Read more</label>
+            <input id="readmore-${category}" type="checkbox"> 
+            <div class="cc-more-info">{{category${capitalizeFirstLetter(category)}}}</div>
+        </li>`
+      ).join("")
+      + '</ul>',
+    policiesLinks: `<div class="cc-policies-links">For more information, please see our 
+      <a class="cc-link" href="{{cookiePolicyLink}}" rel="noopener noreferrer nofollow" target="_blank">Cookie Policy</a> / 
```

Comment:
For internal navigation, there is no need to clear the referrer. As a user of the library, I would like to be able to specify if the `noreferrer` flag should be used or not.